pom.xml

```diff
@@ -357,6 +357,13 @@
                 <gpg.passphrase>${env.GPG_PASSPHRASE}</gpg.passphrase>
             </properties>
         </profile>
+        <profile>
+            <id>java17</id>
+            <properties>
+                <!-- For java 17, we need these jvm args when running tests due to errors caused by other libraries. -->
+                <surefireJvmArgs>--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.logging/java.util.logging=ALL-UNNAMED</surefireJvmArgs>
```

Comment:
For Java17, we'll use the various --add-opens JVM args to allow tests to run clean